assets/logos/alerting-drivers.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Actually, I do not know how the logo is handled before. 
I saw this [PR](https://github.com/rancher/charts/pull/1215) put the logo to the dev-v2.6 branch, but this [PR](https://github.com/rancher/charts/pull/1019) put the logo to the release-v2.6 branch.

Reply:
Yeah, so before we needed to commit it directly into the release/v2.5 branch since the sync process did not take over assets which aren't charts. I think now we can just commit it into dev-v2.5 and port it over on a release.

Reply:
But the url in the chart.yaml is still pointing to the file in the dev-v2.5 branch unless we will correct it when merging into the release branch.

Reply:
The icon url should point towards the release branch, so I suppose we have to go straight to release with logo. I told manuel to do it opposite way so we'll need to move SRIOV one also.

Reply:
The url is updated and it points to the release link now:
```
https://charts.rancher.io/assets/logos/alerting-drivers.svg
```